assets/src/stories-editor/constants.js

```diff
@@ -116,17 +116,17 @@ export const MAX_IMAGE_SIZE_SLUG = 'amp_story_page';
 
 export const ANIMATION_DURATION_DEFAULTS = {
 	drop: 1600,
-	'fade-in': 500,
-	'fly-in-bottom': 500,
-	'fly-in-left': 500,
-	'fly-in-right': 500,
-	'fly-in-top': 500,
-	pulse: 500,
-	'rotate-in-left': 700,
-	'rotate-in-right': 700,
+	'fade-in': 400,
+	'fly-in-bottom': 400,
+	'fly-in-left': 400,
+	'fly-in-right': 400,
+	'fly-in-top': 400,
+	pulse: 400,
+	'rotate-in-left': 600,
+	'rotate-in-right': 600,
 	'twirl-in': 1000,
-	'whoosh-in-left': 500,
-	'whoosh-in-right': 500,
+	'whoosh-in-left': 400,
+	'whoosh-in-right': 400,
 	'pan-left': 1000,
```

Comment:
^ These were originally taken from the [AMP defaults](https://amp.dev/documentation/guides-and-tutorials/start/visual_story/animating_elements/), any reason to change that?

Reply:
The defaults changed. See commit https://github.com/ampproject/amp-wp/pull/3104/commits/814f609c0b825f11d38e2bcce4891779480ae885 and upstream change https://github.com/ampproject/amphtml/pull/21446.

Reply:
Ah, good to know, thanks!